tests/testutils/kubeutils/kind_cluster.go

```diff
@@ -153,28 +153,40 @@ func (k KindCluster) Apply(manifests string) (stdOut, stdErr bytes.Buffer, err e
 }
 
 func (k KindCluster) runKubectl(stdin io.Reader, args ...string) (stdOut, stdErr bytes.Buffer, err error) {
```

Comment:
It sounds like those changes are independent and can be peeled off as a separate PR to come in quickly. Not a big deal overall.

Reply:
yeah they center around a bug I found writing these tests so threw them in the debugging process. I can break them out if you'd prefer but there's only one other consuming test.

Reply:
no, it's fine to have them here since they help with the PR, just conveying that in general having smaller PRs will help